digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -627,6 +627,24 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
     }
   }
 
+  def updateProjectLinkGeometry(projectId:Long, username:String, onlyNotHandled:Boolean =false): Unit = {
+    withDynTransaction {
+      val projectLinks= if(onlyNotHandled) ProjectDAO.getProjectLinks(projectId).filter(x=>x.status==LinkStatus.UnChanged) else ProjectDAO.getProjectLinks(projectId)
```

Comment:
if(onlyNotHandled) 
....filter(x=>x.status==LinkStatus.UnChanged)
shouldnt it be 
....filter(x=>x.status==LinkStatus.NotHandled)
 instead ?